tensorflow/lite/kernels/internal/common.h

```diff
@@ -375,55 +375,80 @@ inline Integer FloorLog2(Integer n) {
   }
 }
 
-// The size of the LUT depends on the type of input. For int8 inputs a simple
-// 256 entries LUT is used. For int16 inputs the high 9 bits are used for
+// The size of the LUT depends on the type of input. For uint8 and int8 inputs a
+// simple 256 entries LUT is used. For int16 inputs the high 9 bits are used for
 // indexing and the 7 remaining bits are used for interpolation. We thus use a
 // 513-entries LUT for int16 cases, 512 for the 9-bit indexing and 1 extra entry
 // to interpolate the last value.
-template <typename LutInT>
-constexpr int lut_size() {
-  static_assert(std::is_same<LutInT, int8_t>::value ||
-                    std::is_same<LutInT, int16_t>::value,
-                "Only LUTs with int8 or int16 inputs are supported.");
-  return std::is_same<LutInT, int8_t>::value ? 256 : 513;
+template <typename T>
+constexpr int LUTSize() {
+  static_assert(std::is_same<T, uint8_t>::value ||
+                    std::is_same<T, int8_t>::value ||
+                    std::is_same<T, int16_t>::value,
+                "Only LUTs with uint8, int8 or int16 inputs are supported.");
+  if (std::is_same<T, uint8_t>::value || std::is_same<T, int8_t>::value) {
+    return 256;
+  } else {
+    return 513;
```

Comment:
could you add a comment for describing the meaning of this number?

Reply:
See above